### ISSUE TITLE
Design enhancement - improve metadata documentation

### ISSUE THREAD

**Issue by juliev42** [bio-learn/biolearn#44]
The GEO data access through this platform is awesome! My main request is that the metadata documentation should be improved. It would be helpful to know the metadata available for each GEO dataset without loading it, and the variable encoding should be improved (e.g. it's not clear which of Male vs Female is 1 vs 2).

**Comment by sarudak** [comment]
Yes that makes sense. Getting better access to what metadata is available programatically would be helpful. For sex 1 is female and 2 is male, I agree it's not clear and perhaps I should have gone with text. Until we get better support for metadata here is the source yaml in case you find it useful: https://github.com/bio-learn/biolearn/blob/master/biolearn/data/library.yaml
